ooniapi/services/oonirun/src/oonirun/routers/v2.py

```diff
@@ -33,16 +37,26 @@
 def utcnow_seconds():
     return datetime.now(timezone.utc).replace(microsecond=0)
 
+class OonirunMeta(BaseModel):
+    run_type : str = Field(description="Run type", pattern="^(timed|manual)$")
+    is_charging : bool = Field(description="If the probe is charging")
+    probe_asn : str = Field(pattern=r"^([a-zA-Z0-9]+)$")
```

Comment:
This pattern seems a bit too lax. I would say it should be: `^(AS)?([0-9]{1,10})$`